packages/shared/src/components/layout/common.tsx

```diff
@@ -123,6 +135,15 @@ export const SearchControlHeader = ({
         onChange={(_, index) => setSelectedPeriod(index)}
       />
     ) : null,
+    feedsWithActions.includes(feedName as SharedFeedPage) ? (
+      <ToggleClickbaitShield
+        {...actionBtnProps}
+        origin={
+          feedName === SharedFeedPage.Custom ? Origin.CustomFeed : Origin.Feed
+        }
+        key="toggle-clickbait-shield"
+      />
+    ) : null,
```

Comment:
Is it necessary for us to return null?
```suggestion
    feedsWithActions.includes(feedName as SharedFeedPage) && (
      <ToggleClickbaitShield
        {...actionBtnProps}
        origin={
          feedName === SharedFeedPage.Custom ? Origin.CustomFeed : Origin.Feed
        }
        key="toggle-clickbait-shield"
      />
    ),
```